packages/navigation/src/UIPagerView/UIPagerViewContainer.tsx

```diff
@@ -337,18 +339,15 @@ const styles = StyleSheet.create({
         height: 1,
     },
     fixedTabBar: {
-        height: UIConstant.pagerViewHeight,
+        height: UIConstant.pager.pagerViewHeight,
         backgroundColor: 'transparent',
         shadowColor: 'transparent',
         elevation: 0,
         justifyContent: 'center',
+        bottom: UIConstant.pager.tabBarOffset,
+        borderWidth: 0,
```

Comment:
@savelichalex This is the core of the fix. The rest of the styles are just to improve the look.

Reply:
Did you manage to understand how it works? Because I don't quite understand how it fixes the problem :)

Reply:
@savelichalex This is just a hack, I'm not sure why it works at all :)

Reply:
Please leave a comment then with breadcrumbs with 1. what was the problem 2. how to reproduce it 3. how it fixes it (here you can write that there is some magic behind)